docs/discussion/values/technical-values.rst

```diff
@@ -0,0 +1,67 @@
+Technical Values
+=================
+
+Technical values describe our processes and design principles. They are
+important to guide the trajectory of ``ppb`` as a software tool with a
+sustainable development lifetime.
```

Comment:
Do we have non-design technical values?

Reply:
The big one I want to get in here at some point is some spin on "Let's try" with discussion about the person doing the work on a feature gets extra leeway in how that piece works (IE, the parts not bound by other principles.)